packages/llvm6.rb

```diff
@@ -0,0 +1,313 @@
+require 'package'
+
+class Llvm6 < Package
+  description 'The LLVM Project (version 6.0.0) is a collection of modular and reusable compiler and toolchain technologies. The optional packages including clang, lld, polly, compiler-rt, libcxx, libcxxabi and openmp are included.'
+  homepage 'http://llvm.org/'
+  version '6.0.0'
+  source_url 'http://llvm.org/releases/6.0.0/llvm-6.0.0.src.tar.xz'
+  source_sha256 '1ff53c915b4e761ef400b803f07261ade637b0c269d99569f18040f3dcee4408'
+
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/llvm6-6.0.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/llvm6-6.0.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/llvm6-6.0.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/llvm6-6.0.0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '32928186027577aee2618dc4826b09900de5842eac4d5dc82743a917f9fcaae1',
+     armv7l: '32928186027577aee2618dc4826b09900de5842eac4d5dc82743a917f9fcaae1',
+       i686: 'bbd3d9b82dfc1a98740eef4c784407caa9aa7c7f1d183374b11962ba5fdebcb6',
+     x86_64: '664c18423806dbd1987e3e81c64b44cf6f447a671aae46eeb25cbcba597d714e',
+  })
+
+  depends_on 'python27' => :build # for test suite
+  depends_on 'icu4c'              # fix uconv.h missing
+  depends_on 'libtirpc'
+  
+  def self.preinstall
+    
+    ############################# Download clang (tools) ###########################################
+    url_clang = 'http://llvm.org/releases/6.0.0/cfe-6.0.0.src.tar.xz'
+    uri_clang = URI.parse url_clang
+    filename_clang = File.basename(uri_clang.path)
+    sha256sum_clang = 'e07d6dd8d9ef196cfc8e8bb131cbd6a2ed0b1caf1715f9d05b0f0eeaddb6df32'
+    
+    if File.exist?(filename_clang) && Digest::SHA256.hexdigest( File.read("./#{filename_clang}") ) == sha256sum_clang
+      puts "Clang soruce code exists and perform unpacking".lightred
```

Comment:
Can we make the color yellow instead of red?  Red generally indicates an error.  Also, soruce is misspelled.